docs/explanations/architecture-2.md

```diff
@@ -130,8 +132,8 @@ The Kubewarden consists of these components:
   WebAssembly modules have detailed documentation in the
   [writing policies](../tutorials/writing-policies/index.md) sections.
 
-- [Policy server](https://github.com/kubewarden/policy-server):
-  The policy server receives requests for validation.
+- [PolicyServer](https://github.com/kubewarden/policy-server):
```

Comment:
```suggestion
- [Policy server](https://github.com/kubewarden/policy-server):
```

Oof this is difficult. This refers to the server, a policy-server, which receives requests. It isn't the CR PolicyServer.